Marlin/language.h

```diff
@@ -65,13 +68,27 @@
   #define MACHINE_NAME "HEPHESTOS"
   #undef FIRMWARE_URL
   #define FIRMWARE_URL "http://www.bq.com/gb/downloads-prusa-i3-hephestos.html"
-#else // Default firmware set to Mendel
+#elif MB(BRAINWAVE_PRO)
+  #define MACHINE_NAME "Kossel Pro"
+  #ifndef FIRMWARE_URL
+    #define FIRMWARE_URL "https://github.com/OpenBeamUSA/Marlin/"
+  #endif
+#else
+  #ifdef CUSTOM_MENDEL_NAME
+    #define MACHINE_NAME CUSTOM_MENDEL_NAME
+  #endif
+#endif
+
+#ifndef MACHINE_NAME
   #define MACHINE_NAME "Mendel"
 #endif
 
-#ifdef CUSTOM_MENDEL_NAME
-  #undef MACHINE_NAME
-  #define MACHINE_NAME CUSTOM_MENDEL_NAME
```

Comment:
So, rather than a custom Mendel name, it is meant to be a custom machine name.
Therefore, should we not call it CUSTOM_MACHINE_NAME?